.github/workflows/on-misskey-release.yml

```
@@ -0,0 +1,50 @@
+name: On Misskey Release (Update Release Notes)
+
+on:
+  repository_dispatch:
+    types: [misskey_release]
+
+jobs:
+  update-release-notes:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      
+      - name: Setup pnpm
+        uses: pnpm/action-setup@v4
+  
+      - name: Setup Node.js
+        uses: actions/setup-node@v4
+        with:
+          cache: 'pnpm'
+
+      - run: corepack enable
+
+      - run: pnpm install
+        working-directory: scripts/changelog-auto-updater
+      
+      - run: pnpm start
+        working-directory: scripts/changelog-auto-updater
+
+      - name: Check diff
+        id: diff
+        run: |
+          git add -N .
+          git diff --name-only --exit-code
+        continue-on-error: true
+
+      - name: Commit changes
+        if: steps.diff.outcome == 'failure'
+        run: |
+          set -x
+          git checkout -b update-release-notes
+          git config user.name github-actions[bot]
+          git config user.email 41898282+github-actions[bot]@users.noreply.github.com
+          git add .
+          git commit -m "docs: Update release notes"
+          git push
+
+      - name: Create Pull Request
+        if: steps.diff.outcome == 'failure'
```

```suggestion
        if: steps.diff.outcome == 'failure'
        env:
          GH_TOKEN: ${{ github.token }}
```

ないと権限エラーになりがち

https://cli.github.com/manual/gh_help_environment